webapp/greenhouse.py

```diff
@@ -177,3 +179,16 @@ def submit_application(
         )
 
         return response
+
+    # Get list of external departments from the Harvest API
+    def get_departments(self):
+        content_url = "https://harvest.greenhouse.io/v1///custom_field/155450"
+        key = harvest_api_key + ":"
```

Comment:
Can you add a comment here to explain why you are adding a `:` to the key? Something like, "The key is passed as the username with a blank password. Therefore, the key needs an appended colon to achieve this"